airflow/utils/context.py

```diff
@@ -166,6 +166,44 @@ class OutletEventAccessor:
     """
 
     extra: dict[str, Any]
+    _raw_key: str | Dataset | DatasetAlias
+    dataset_alias_event: DatasetAliasEvent | None
+
+    def __init__(
+        self,
+        extra,
+        *,
+        raw_key: str | Dataset | DatasetAlias,
+        dataset_alias_event: DatasetAliasEvent | None = None,
+    ) -> None:
+        self.extra = extra if extra else {}
+        self._raw_key = raw_key
+        self.dataset_alias_event = dataset_alias_event
+
+    def __str__(self) -> str:
+        return f"OutletEventAccessor(_raw_key={self._raw_key}, extra={self.extra}, dataset_alias_event={self.dataset_alias_event})"
```

Comment:
What’s wrong with the default `__str__` format from attrs?

Reply:
Just checked. This should be removed.

Reply:
removed